docs/tutorial.md

```diff
@@ -349,6 +349,137 @@ If the deployment is complete, `plan.status` in the returned JSON
 should equal "COMPLETE".  If it is, you should see two `meta-data`
 tasks and three `data` tasks running under the service.
 
+# Plan management
+As noted earlier, stateful services often have particular deployment 
+requirements.  The SDK organizes deployment of tasks by means of Plans. 
+Plans are composed of Phases which are composed of Blocks.  A Block 
+encapsulates the smallest element of work executed in a Plan.  Phases
```

Comment:
Might be instructive to go ahead and relate a block to a task here.  Since conceptually block == task